--- easytag-2.1/easytag.desktop	2007-10-16 16:36:17.000000000 +0000
+++ easytag-2.1/easytag.desktop	2007-10-16 16:36:23.000000000 +0000
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Name=EasyTAG
 GenericName=Audio File Tag Viewer/Editor
+GenericName[pl]=Edytor etykiet plików audio
 Comment=EasyTAG is an utility for viewing and editing tags for MP3, MP2, FLAC, Ogg Vorbis, MusePack and Monkey's Audio files, with a GTK+ GUI.
 Comment[cs]=EasyTAG je utilita na prohlížení, editaci a zápis tagů do vašich MP3, MP2, FLAC, Ogg Vorbis, MusePack a Monkey Audio souborů.
 Comment[da]=Et program til at vise/redigere MP3 og Ogg Vorbis tags med et GTK+ interface.
@@ -20,6 +22,6 @@
 Icon=EasyTAG_icon
 Terminal=false
 Type=Application
-Categories=AudioVideo;AudioVideoEditing;GTK;
+Categories=AudioVideo;AudioVideoEditing;GTK;Audio;
 X-Desktop-File-Install-Version=0.14
 MimeType=audio/x-mp3;application/ogg;x-directory/normal;
