--- configure.in~	Tue Dec 31 20:36:47 2002
+++ configure.in	Tue Dec 31 20:36:39 2002
@@ -141,7 +141,7 @@
 main ()
 {
     FILE *output;
-    dnl output=fopen("conftest.id3","w"); dnl doesn't work on NetBSD.
+	#output=fopen("conftest.id3","w"); dnl doesn't work on NetBSD.
     output=fopen("/tmp/conftest.id3","w");
     fprintf(output,"ID3LIB_MAJOR=%d\nID3LIB_MINOR=%d\nID3LIB_PATCH=%d\n",ID3LIB_MAJOR_VERSION,ID3LIB_MINOR_VERSION,ID3LIB_PATCH_VERSION);
     fclose(output);
