--- easytag-0.30/easytag.desktop~	Tue Jul 15 20:55:54 2003
+++ easytag-0.30/easytag.desktop	Tue Jul 15 21:03:32 2003
@@ -7,13 +7,18 @@
 Name[ja]=EasyTAG
 Name[da]=EasyTAG
 Name[nl]=EasyTAG
+Name[pl]=EasyTAG
 Name[ru]=EasyTAG
 Name[sv]=EasyTAG
-Comment=EasyTAG is an utility for viewing and editing tags for MP3, MP2, FLAC, Ogg Vorbis, MusePack and Monkey's Audio files, with a GTK+ GUI.
-Comment[fr]=EasyTAG est un utilitaire pour visualiser et Èditer les tags des fichiers MP3, MP2, FLAC, Ogg Vorbis, MusePack et Monkey's Audio avec une interface GTK+.
-Comment[ja]=GTK+ »« MP3/Ogg Vorbis •ø•∞§Œ…Ωº®/ ‘Ω∏•ƒ°º•Î
-Comment[da]=Et program til at vise/redigere MP3 og Ogg Vorbis tags med et GTK+ interface.
+Comment=EasyTAG is an utility for viewing and editing tags for MP3, MP2, FLAC, Ogg Vorbis, MusePack and Monkey's Audio files, with a GTK+ GUI
+Comment[da]=Et program til at vise/redigere MP3 og Ogg Vorbis tags med et GTK+ interface
+Comment[fr]=EasyTAG est un utilitaire pour visualiser et √©diter les tags des fichiers MP3, MP2, FLAC, Ogg Vorbis, MusePack et Monkey's Audio avec une interface GTK+
+Comment[ja]=GTK+ Áâà MP3/Ogg Vorbis „Çø„Ç∞„ÅÆË°®Á§∫/Á∑®ÈõÜ„ÉÑ„Éº„É´
+Comment[pl]=Oparty o GTK+ program do przeglƒÖdania/edycji znacznik√≥w plikÛw MP3, MP2, FLAC, Ogg Vorbis, MusePacka i Audio Monkeya
 Exec=easytag
 Icon=EasyTAG.xpm
-Terminal=0
+Terminal=false
 Type=Application
+Categories=GTK;Multimedia
+Encoding=UTF-8
+# vi: encoding=utf-8
