--- easytag-0.30/easytag.desktop~	Tue Jul 15 20:55:54 2003
+++ easytag-0.30/easytag.desktop	Tue Jul 15 21:03:32 2003
@@ -7,13 +7,18 @@
 Name[ja]=EasyTAG
 Name[da]=EasyTAG
 Name[nl]=EasyTAG
+Name[pl]=EasyTAG
 Name[ru]=EasyTAG
 Name[sv]=EasyTAG
 Comment=An utility for viewing/editing MP3 and OGG tags with a GTK+ GUI.
-Comment[fr]=Un utilitaire pour visualiser et Èditer les tags des fichiers MP3 et OGG ‡ l'aide d'une interface GTK+.
-Comment[ja]=GTK+ »« MP3/OGG •ø•∞§Œ…Ωº®/ ‘Ω∏•ƒ°º•Î
-Comment[da]=Et program til at vise/redigere MP3 og OGG tags med et GTK+ interface.
+Comment[da]=Et program til at vise/redigere MP3 og OGG tags med et GTK+ interface
+Comment[fr]=Un utilitaire pour visualiser et √©diter les tags des fichiers MP3 et OGG √† l'aide une interface GTK+
+Comment[ja]=GTK+ Áâà MP3/OGG „Çø„Ç∞„ÅÆË°®Á§∫/Á∑®ÈõÜ„ÉÑ„Éº„É´
+Comment[pl]=Oparty o GTK+ program do przeglƒÖdania/edycji MP3 oraz znacznik√≥w OGG
 Exec=easytag
 Icon=EasyTAG.xpm
-Terminal=0
+Terminal=false
 Type=Application
+Categories=GTK;Multimedia
+Encoding=UTF-8
+# vi: encoding=utf-8
