--- easytag-0.31_gtk2.4_pre3/easytag.desktop.wiget	2004-08-30 21:04:45.000000000 +0200
+++ easytag-0.31_gtk2.4_pre3/easytag.desktop	2004-10-17 20:23:02.487482938 +0200
@@ -20,6 +20,6 @@
 Terminal=false
 Type=Application
 Encoding=UTF-8
-Categories=Application;AudioVideo;AudioVideoEditing;
+Categories=Audio;AudioVideoEditing;
 X-Desktop-File-Install-Version=0.3
 MimeType=x-directory/normal;audio/x-mp3;application/ogg;
