diff -Nru easytag-0.20/configure.in easytag-0.20-new/configure.in
--- easytag-0.20/configure.in	Fri May 10 12:28:42 2002
+++ easytag-0.20-new/configure.in	Tue Oct 15 00:08:46 2002
@@ -88,7 +88,7 @@
     fclose(output);
     exit(0);
 }
-], . conftest.id3; echo "${ID3LIB_MAJOR}.${ID3LIB_MINOR}.${ID3LIB_PATCH}", AC_MSG_WARN(could not determine id3lib version),[echo $ac_n "cross compiling; assuming ${ID3LIB_MAJOR}.${ID3LIB_MINOR}.${ID3LIB_PATCH} $ac_c"])
+], . ./conftest.id3; echo "${ID3LIB_MAJOR}.${ID3LIB_MINOR}.${ID3LIB_PATCH}", AC_MSG_WARN(could not determine id3lib version),[echo $ac_n "cross compiling; assuming ${ID3LIB_MAJOR}.${ID3LIB_MINOR}.${ID3LIB_PATCH} $ac_c"])
 AC_DEFINE_UNQUOTED(ID3LIB_MAJOR, $ID3LIB_MAJOR)
 AC_DEFINE_UNQUOTED(ID3LIB_MINOR, $ID3LIB_MINOR)
 AC_DEFINE_UNQUOTED(ID3LIB_PATCH, $ID3LIB_PATCH)
